config-secret/file-system/src/main/resources/application.properties

```diff
@@ -1,6 +1,7 @@
 quarkus.openshift.expose=true
 quarkus.openshift.secret-volumes.app-config.secret-name=app-config
 quarkus.openshift.mounts.app-config.path=/deployments/config
+quarkus.openshift.env.vars.smallrye-config-locations=/deployments/config
```

Comment:
Please add a TODO to remove the workaround with the link to the issue so we don't keep this in testsuite